src/components/NavBar.tsx

```diff
@@ -113,14 +114,23 @@ const NavBar = (): JSX.Element => {
   const router = useRouter();
   const classes = useStyles();
 
-  const fellowItems = [
-    { title: "2024 Cohort", url: "/fellows" },
-    { title: "Showcase", url: "/showcase" },
-  ];
-
   const aboutItems = [
+    { title: "Project", url: "/project" },
+    { title: "Core Team", url: "/team" },
     { title: "Advisory", url: "/advisory" },
-    { title: "SDOH & Place Project", url: "/about" },
+  ];
+
+  const resourcesItems = [
+    //{ title: "Data Discovery", url: "/search" },
+    { title: "Community Toolkit", url: "https://toolkit.sdohplace.org", target: '_blank' },
+    //{ title: "SDOH Guides", url: "/guides" },
+  ];
+
+  const communityItems = [
+    { title: "Fellows", url: "/fellows" },
+    { title: "Showcase", url: "/showcase" },
+    //{ title: "Place Mini-Projects", url: "/mini-projects" },
+    { title: "Things We Like!", url: "/recommendations" },
```

Comment:
We may want to comment this out of the Navbar before merging:
```suggestion
    //{ title: "Things We Like!", url: "/recommendations" },
```

Unless we have contents to add for this page